FFMpegCore/FFMPEG/Argument/Atoms/ForceFormatArgument.cs

```diff
@@ -5,16 +5,17 @@ namespace FFMpegCore.FFMPEG.Argument
     /// <summary>
     /// Represents force format parameter
     /// </summary>
-    public class ForceFormatArgument : Argument<VideoCodec>
+    public class ForceFormatArgument : Argument<string>
```

Comment:
Why this change? An enum makes it much clearer what options are available, to the user

Reply:
I think WeihanLi's flexible RawArgument would be a better solution than to untype existing arguments?

Reply:
FFMpeg contains much more video codecs and containers then declared in `VideoCodec` enum. And their quantity differs between builds and ffmpeg modules selected for compilation. Users still can use `VideoCodec` enum to instatiate new `ForceFormatArgument` and also use string values. So we can use `ForceFormatArgument("matroska")` and  `ForceFormatArgument(VideoCodec.MpegTs)`.

Definig enums is very user friendly, however you are ruining flexibility of ffmpeg by restricting users to use only predefined sets of codecs.

And what about #49. It is very neccesary argument for a lot of cases but not this. If we force users to use `CustomArgument("-f matroska")` instead of `ForceFormatArgument("matroska")` it would be a kind of workaround or hack around restictions that are provided by only using enum for `ForceFormatArgument`.

More over, we should remember that just a single argument of the same type can by added to `ArgumentContainer` cause it `Dictionary<Type, Argument>` under the hood.